apps/site/authors.json

```diff
@@ -14,6 +14,11 @@
     "name": "Augustin Mauroy",
     "website": "https://github.com/AugustinMauroy"
   },
+  "Aviv Keller": {
```

Comment:
no needed it's only for blog

Reply:
No, it's also for learn. My name doesn't show up otherwise